Makefile

```diff
@@ -20,7 +20,7 @@ provider::
 
 build_nodejs:: VERSION := $(shell pulumictl get version --language javascript)
 build_nodejs::
-	rm -rf nodejs/eks/bin
+	rm -rf nodejs/eks/bin/*
```

Comment:
Otherwise, deleting the whole `bin` dir was preventing subsequent `yarn link` calls in the `install_nodejs_sdk` target from succeeding after a prior `yarn link`.